src/main/resources/application-prod.yml

```
@@ -0,0 +1,54 @@
+# PROD
+
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+
+  # JPA
+  jpa:
+    hibernate:
+      ddl-auto: none
+    database: postgresql
+    show-sql: false
+    properties:
+      hibernate:
+        format_sql: false
+        use_sql_comments: false
+        highlight_sql: false
+        generate_statistics: false
+        cache:
+          use_second_level_cache: false
+      open-in-view: false
+
+# logging
+logging:
+  level:
+    root: info
+    com.sprint.mission.discodeit: info
+    org:
+      hibernate:
+        sql: debug
+
+# swagger
+springdoc:
+  api-docs:
+    path: /api-docs     # OpenAPI 명세서의 엔드포인트 설정 (기본값: /v3/api-docs)
+  default-consumes-media-type: application/json   # 기본 요청 Content-Type
+  default-produces-media-type: application/json   # 기본 응답 Content-Type
+  swagger-ui:
+    operations-sorter: alpha    # API 메서드 알파벳 순 정렬
+    tags-sorter: alpha          # API 태그 알파벳 순 정렬
+    path: /swagger-ui
+    display-request-duration: true
+    doc-expansion: none
+    filter: true
+    show-extensions: true
+
+# port
+server:
+  port: ${APP_PORT:80}
```

```suggestion
server:
  port: 80
```

SERVER_PORT 리눅스 환경변수로 충분히 대체할 수 있을거 같네요!

그리고 과제 요구사항과 별개로 말씀 드리면,,, 80번 포트는 보통 root 권한을 가진 상태에서 점유할 수 있어요.
근데 컨테이너 보안을 위해 실무에선 root 권한으로 프로세스를 실행하지 않습니다.
80번 포트는 잘 사용하지 않고 8080을 사용하고, 외부에 프록시나 로드밸런서를 통해 80번 포트를 열어두곤 합니다. 참고해주세요!